.eslintrc.json

```diff
@@ -19,6 +19,7 @@
         "react"
     ],
     "rules": {
+      "react/prop-types": "off"
```

Comment:
Why turn that off?

Reply:
It threw errors everytime we tried to de-structure props.